tensorflow/lite/micro/kernels/conv_test_common.cc

```diff
@@ -220,5 +220,23 @@ TfLiteStatus TestConvQuantizedPerChannel(
       conv_params, registration, output_data);
 }
 
+TfLiteStatus TestConvQuantizedPerChannel(
```

Comment:
Could you please add some comments here? something along the line like "Test XXXX with 16 bit activation, 8 bit weight, 32 bit accumuator".  I would add similiar comments on the other two previous functions. 

This is a style suggestion and I tend to be on verbose side. Feel free to push back.

Reply:
I agree this isn't very readable as is. Added the suggested comment to all 3 functions.